docs/AboutUs.md

```diff
@@ -5,19 +5,18 @@ title: About Us
 
 We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
-You can reach us at the email `seer[at]comp.nus.edu.sg`
+This project was done by a team of 4 members.
 
 ## Project team
 
-### John Doe
+### Li Hongguang
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/Hongguang.jpeg" width="200px">
```

Comment:
File name should end with .png regardless of format of the file. It also needs to be in lower case.

Reply:
Fixed <3